datafusion-federation/src/schema_cast/record_convert.rs

```diff
@@ -153,4 +175,43 @@ mod test {
         let result = try_cast_to(batch_input(), to_schema()).expect("converted");
         assert_eq!(3, result.num_rows());
     }
+
+    fn large_string_schema() -> SchemaRef {
```

Comment:
```suggestion
    fn large_string_from_schema() -> SchemaRef {
```

nit: consistency with the below